test-cases/gutenberg/inserter.md

```diff
@@ -0,0 +1,11 @@
+# Block Inserter - Test Cases
+
+##### TC0001
+
+**Blocks are sorted properly in the inserter**
+
+**Steps**
+1. Open the block inserter
+
+**Expected Behavior**
+Core blocks should be listed before non-core blocks in [the order that they are registered](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/index.native.js#L234-L270).
```

Comment:
Regarding the link, I think we should reference the file `https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/index.native.js` and probably the function `registerCoreBlocks ` but not the lines because in the future when adding more blocks it will get outdated.

Reply:
👍  Sure, updated the link to remove the line numbers.

Reply:
I think we should also add a reference that the list can be found inside the `registerCoreBlocks`, wdyt?

Reply:
👍 Added a reference to the function.